release-please-config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "release-type": "java-yoshi",
+  "versioning": "always-bump-minor",
+  "separate-pull-requests": false,
+  "include-component-in-tag": false,
+  "packages": {
+    ".": {
+      "component": "main"
+    }
+  },
+  "plugins": ["maven-workspace"]
```

Comment:
You should skip this plugin if there's only a single component unless you have pom.xml files without annotations.

It shouldn't hurt but will make extra API calls and do extra work to build the dependency tree.

Reply:
@meltsufin, I remember you fixed all the pom.xml's to have the annotations. Should I remove this plugin then? or better to have it around for now?

Reply:
All poms are fixed to have annotations and there is a check to verify it.